backend/user_page/models/DeveloperMessage.py

```diff
@@ -0,0 +1,10 @@
+from django.db import models
+
+from backend.user_page.models.User import User
+
+
+class DeveloperMessage(models.Model):
+    id = models.AutoField(primary_key=True)
+    message = models.CharField(max_lenght=500)
+    timestamp = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
The user id must have the possibility of being null, in case the message comes from a user that does not have an account. This is specified in the diagram with the "N" (nullable) flag. Also, I do not think we need to have the messages deleted by cascade when a user is deleted.